sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -103,7 +103,10 @@ public class DataLakeFileSystemAsyncClient {
      */
     public static final String ROOT_FILESYSTEM_NAME = "$root";
 
-    private static final String ROOT_DIRECTORY_NAME = "";
+    /**
+     * Special directory name for the root directory of the file system.
```

Comment:
Would include the message you included in the javadoc for `getRootDirectory`

```suggestion
     * Special directory name for the root directory of the file system.
     * <p>
     * This should only be used while getting the root directory from the file system client.
```